src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4073,7 +4073,7 @@ void M()
     {
         int a = 1;
         int {|Rename:y1|} = C.y;
-        var t = (a, x: y1);
+        var t = (a: a, x: y1);
```

Comment:
@CyrusNajmabadi What did you think about this? I find it surprising that my preferences would cause refactorings to not simplify nodes produced by complexification.